package.json

```diff
@@ -125,6 +125,7 @@
 		"babel-plugin-require-context-hook": "1.0.0",
 		"benchmark": "2.1.4",
```

Comment:
I think the new `@wordpress/e2e-selenium` package needs to be listed as a dependency in here? As above with the other `@wordpress/` dependencies.

I'm assuming _all_ of our packages should be here. I've personally encountered some issues otherwise, especially with regards to dependencies and having an accurate `package-lock.json` ([example](https://github.com/WordPress/gutenberg/pull/21221/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)).

Reply:
Done.